hummingbot/client/command/config_command.py

```diff
@@ -207,10 +207,6 @@ def get_all_available_config_keys() -> List[str]:
             self._notify('Invalid choice. Please enter "create" or "import".')
             strategy_path = await self._import_or_create_strategy_config()
 
-        # Validate response
```

Comment:
Why is this deleted?

Reply:
@martinkou I realized that for this particular prompt, the answer is validated twice -- at line 234 in the same file. Normally this is ok. But in this particular case, since we try to re-load and rewrite the config file after a set of strategy config is selected, it causes an async bug